splunk/resource_splunk_authorization_roles_test.go

```diff
@@ -47,8 +49,6 @@ func TestAccSplunkAuthorizationRoles(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "imported_roles.#", "1"),
 					resource.TestCheckResourceAttr(resourceName, "imported_roles.0", "user"),
 					resource.TestCheckResourceAttr(resourceName, "capabilities.#", "2"),
-					resource.TestCheckResourceAttr(resourceName, "capabilities.0", "accelerate_datamodel"),
-					resource.TestCheckResourceAttr(resourceName, "capabilities.1", "change_authentication"),
```

Comment:
It's not (easily) possible to test the values of a `TypeSet` field, so this had to be removed.

Terraform's test framework effectively tests it for you, however, since it applies the configuration and checks for a non-empty plan afterwards.  So as long as `capabilities` are defined in the configuration being tested, there is implicit testing of the values of the `capabilities` stored in state.